### ISSUE TITLE
Can we drop the "webappsec-" from the rep name

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/webappsec-credential-management#229]
Makes the repo hard to find :( Plus specs move to different working groups all the time.

**Comment by clelland** [comment]
We might want to do that with all of the WebAppSecWG repos, if we do it here. I think it's common across all of them.

**Comment by marcoscaceres** [comment]
Yeah, that would be great. I'll see if we can get that added to the WG agenda.

**Comment by marcoscaceres** [comment]
Filed https://github.com/w3c/webappsec/issues/652
